source/quakedef.h

```diff
@@ -42,6 +42,8 @@ extern int psp_system_model;
 #define	WINQUAKE_VERSION	0.996
 #define	LINUX_VERSION		1.30
 #define	X11_VERSION			1.10
+#define WIIGX_VERSION			0.09
+#define QUAKE_WII_BASEDIR "/apps/nzportable"
```

Comment:
This should go in the main platform source file for gxQuake, not in quakedef ideally